### PR TITLE
Add SLA auditor container, clean up plugins

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

* Add sla auditor discovery plugin
* Clean up other plugin launch system

### CODE REVIEW

**Review by alecsavvy — APPROVED** [review]

File: audius-cli [hunk 1]
```diff
@@ -414,9 +357,9 @@ def launch(ctx, service, seed, chain, yes):
         check=True,
     )
 
-    launch_plugins(ctx)
     if service == "discovery-provider":
-        ctx.invoke(launch_required_plugins)
+        ctx.invoke(launch_chain)
+        ctx.invoke(launch_plugins)
```
Comment (alecsavvy) [inline audius-cli:362]: noice

File: discovery-provider/docker-compose.plugins.yml [hunk 2]
```diff
@@ -23,6 +23,43 @@ services:
     ports:
       - "6001:6001"
 
+  notifications:
```
Comment (alecsavvy) [inline discovery-provider/docker-compose.plugins.yml:26]: nice to see all these together

File: discovery-provider/prod.env [hunk 3]
```diff
@@ -23,7 +23,7 @@ audius_elasticsearch_url='http://elasticsearch:9200'
 audius_elasticsearch_search_enabled='true'
 audius_enable_rsyslog=false
 audius_eth_contracts_registry=0xd976d3b4f4e22a238c1A736b6612D22f17b6f64C
-audius_eth_token_address=0x5375BE4c52fA29b26077B0F15ee5254D779676A6
+audius_eth_token_address=0x18aAA7115705e8be94bfFEBDE57Af9BFc265B998
```
Comment (alecsavvy) [inline discovery-provider/prod.env:26]: is this needed? unsure how this is related to the PR?

Comment (raymondjacobson) [reply discovery-provider/prod.env:26]: it is, in order to init the eth contracts right (and reference the token), this value needs to be correct :)

**Review by alecsavvy — APPROVED** [review]
thanks for the plugins update!

**Pull request merged**